go/api/config_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func TestDefaultStandaloneConfig(t *testing.T) {
-	config := NewRedisClientConfiguration()
+	config := NewGlideClientConfiguration()
```

Comment:
If you want to reduce the size of the PR, you can split renaming and submit it in another PR

Reply:
The renaming is a small change in around 2 existing files only, prefer to keep it here as it won't affect the size of PR much.